docs/editor/artificial-intelligence.md

```diff
@@ -87,11 +79,82 @@ In the image above, Copilot presents three suggestions. You can accept the entir
 
 If you don't want to accept any of the suggestions, you can continue typing, and Copilot will continue providing suggestions as you work.
 
+## Getting the most out of Copilot
+
+There are things you can do to help ("prompt") Copilot give you the best possible suggestions. And the good news is that you are probably already doing right things since they help you and your colleagues understand your code.
+
+Copilot suggestions work best when Copilot has sufficient context to know what you're doing and what you want help with. Just as you would provide a colleague with the context when asking for help with a specific programming task, you can do the same with Copilot.
+
+### Providing context
+
+#### Open files
+
+Copilot looks at the current and open files in your editor to analyze the context and create appropriate suggestions. Having related files open in VS Code while using Copilot helps set this context and lets the Copilot see a bigger picture of your project.
+
+#### Top level comment
+
+Just as you would give a short high level introduction to a coworker, a top level comment in the file you're working in can help Copilot understand the overall context of the pieces you will be creating.
+
+Example of a good top level comment
+
+#### Appropriate includes/module references
+
+It's best to manually set the includes or module references you need for your work. Copilot can make suggestions, but you likely know best what dependencies you'll need to include. This can also help let Copilot know what frameworks, libraries, and their versions you'd like it to use when crafting suggestions.
+
+Example of a Python include
+
+#### Meaningful function/method names
```

Comment:
Maybe just say "function" or "method" to make the header a bit cleaner?

Reply:
I didn't want to leave anyone out but maybe this is showing my C# roots.
Do you think it's find to just same functions and not worry about Java/C#?

Reply:
I think it's okay! And if we see feedback users find this confusing or non-inclusive, we can certainly iterate on it over time.